### PR TITLE
Added Firebase showcase

### PR DESCRIPTION
**Pull request by danielmocan**

### CODE REVIEW

File: showcase/firebase.md [hunk 1]
```diff
@@ -36,9 +33,61 @@ Firestore is a big improvement over Realtime Database. You can find a list of di
 * its an easy to use file hosting, you files will be served from CDNs using http2
```
Comment (alexnm) [inline showcase/firebase.md]: typo: "your files"

File: showcase/firebase.md [hunk 2]
```diff
@@ -36,9 +33,61 @@ Firestore is a big improvement over Realtime Database. You can find a list of di
...
 
 
-### [Pricing](https://firebase.google.com/pricing/)
+### Pricing
 * they have pretty decent free tire plan, you have the option to pay as you go or to choose a monthly subscription
```
Comment (alexnm) [inline showcase/firebase.md]: "free tier plan"

File: showcase/firebase.md (hunk 1 above)
Comment (danielmocan) [reply showcase/firebase.md]: done

File: showcase/firebase.md (hunk 2 above)
Comment (danielmocan) [reply showcase/firebase.md]: done

**Pull request merged**